docs/glossary.md

```diff
@@ -0,0 +1,120 @@
+# Glossary
+
+## Accuracy
+The fraction of predictions a classification model has correctly identified. Formula:
+
+$$
+\text{accuracy} = \frac{\text{True Positives + True Negatives}}{\text{Total amount of data points}}
+$$
+
+## Application Programming Interface (API)
+An API allows independent applications to communicate with each other and exchange data.
+
+## Classification
+Classification refers to dividing a data set into multiple chunks, which are then considered "classes".
+
+## Confusion Matrix
+A confusion matrix is a table that is used to define the performance of a classification algorithm.
+It classifies the predictions to be either be true positive, true negative, false positive or false negative.
+
+## Decision Tree
+A Decision Tree represents the process of conditional evaluation in a tree diagram.
+
+## F1-Score
+The harmonic mean of [precision](#precision) and [recall](#recall). Formula:
```

Comment:
If you use another term defined in the glossary, please link to its section, as done here.

Reply:
Done. Some references I put directly within the LaTeX code... please let me know if this works as intended.

Reply:
You can check the generate page [here](https://safe-ds-stdlib--292.org.readthedocs.build/en/292/glossary/). As you can see there, placing references inside the LaTeX code does **not** work.